cwm_worker_operator/deployments_manager.py

```diff
@@ -168,6 +168,14 @@ def deploy(self, deployment_config, **kwargs):
     def is_ready(self, namespace_name, deployment_type, minimal_check=False):
         return cwm_worker_deployment.deployment.is_ready(namespace_name, deployment_type, minimal_check=minimal_check)
 
+    def get_health(self, namespace_name, deployment_type):
+        return cwm_worker_deployment.deployment.get_health(namespace_name, deployment_type)
+
+    def get_worker_id_namespaces(self):
+        ret, out = subprocess.getstatusoutput("kubectl get ns | tail -n+2 | cut -d' ' -f1")
```

Comment:
Suggested change for command:

```bash
kubectl get ns --no-headers -o=custom-columns=NAME:.metadata.name
```

i.e.

```python
ret, out = subprocess.getstatusoutput('kubectl get ns --no-headers -o=custom-columns=NAME:.metadata.name')
```

Reply:
thanks, changed